blocks/profile-component/profile-component.js

```diff
@@ -0,0 +1,80 @@
+import { getLibs } from '../../scripts/utils.js';
+import '../../components/image-dropzone/image-dropzone.js'
+import { addRepeater, decorateTextfield } from '../../utils/utils.js';
+
+const { createTag } = await import(`${getLibs()}/utils/utils.js`);
+
+function decorateProfileImageDropzone(element){
+    element.classList.add('profile-image');
+    const dropzone = createTag('image-dropzone', {'inputid' : 'alpha-beta'});
+
+    const inputLabel = createTag('div', { slot: 'img-label' , class : 'img-upload-text'});
+    const paragraphs = element.querySelectorAll(':scope > p');
+    paragraphs.forEach((p) => {
+        inputLabel.append(p);
+      });
+    dropzone.append(inputLabel);
+    element.append(dropzone);
+}
+
+function decorateTitle(element){
```

Comment:
todo

Reply:
You can leave // TODO: tags in the code. That'll help us keep track of things directly in the code base.